### PR TITLE
Issue 89 fix

### PR DESCRIPTION
**Pull request by kevinsdooapp**

### CODE REVIEW

**Comment by kevinsdooapp** [comment]
@angelozerr As you said in the issue, the fix which I did for issue 89 covered only Velocity. I'm please to announce you that I have also fix it for Freemarker. I updated the test case Issue89Test to covert Freemarker. Can you review this pull request and merge it?

You'll notice that I've also update the test case Issue93 to help you to reproduce and fix it (since I didn't succeeded myself. Don't forget to uncomment the @Ignore annotation on this test case). 

Thank you in advance!

Kévin

File: document/fr.opensagres.xdocreport.document/src/main/java/fr/opensagres/xdocreport/document/preprocessor/sax/TransformedBufferedDocumentContentHandler.java [hunk 1]
```diff
@@ -487,7 +482,7 @@ public void doEndElement( String uri, String localName, String name )
 
     public FieldMetadata getFieldAsTextStyling( String content )
     {
-        if ( formatter != null && fieldsMetadata != null )
+        if ( formatter != null && fieldsMetadata != null && !SaxUtils.isInstruction(content))
```
Comment (angelozerr) [inline document/fr.opensagres.xdocreport.document/src/main/java/fr/opensagres/xdocreport/document/preprocessor/sax/TransformedBufferedDocumentContentHandler.java]: @kevinsdooapp could you move SaxUtils.isInstruction(content) into IDocumentFormatter#isInstruction please.

**Comment by MaartenVdm** [comment]
What is the status of this issue?
I have the same problem in an if-statement.

**Comment by angelozerr** [comment]
Any PR are welcome!

**Comment by angelozerr** [comment]
Fixed with https://github.com/opensagres/xdocreport/pull/537

**Pull request closed without merging**